nndownload/nndownload.py

```diff
@@ -275,16 +274,16 @@ def parse_datetime_to_timestamp(value) -> int:
 dl_group.add_argument("-c", "--download-comments", action="store_true", dest="download_comments",
                       help="download video comments")
 dl_group.add_argument("--comments-limit", dest="comments_limit", metavar="N", type=int, default=COMMENTS_LIMIT_DEFAULT_N,
-                      help=f"number of comments to download (default: {COMMENTS_LIMIT_DEFAULT_N})")
+                      help=f"number of comments to download per thread (default: {COMMENTS_LIMIT_DEFAULT_N})")
 dl_group.add_argument(
-    "--comments-from", dest="comments_from", type=parse_datetime_to_timestamp, metavar="DATETIME_OR_TIMESTAMP", default=int(datetime.now(timezone.utc).timestamp()), 
```

Comment:
Rather than setting this as runtime, pick it up when we actually begin comments downloading. This also makes it easier to check if a user has actually specified `--coments-from`